src/phpDocumentor/Descriptor/Builder/PhpParser/InterfaceAssembler.php

```diff
@@ -28,11 +28,11 @@ class InterfaceAssembler extends AssemblerAbstract
      *
      * @param Interface_ $data
      *
-     * @return InterfaceDescriptor
+     * @return Interface_
      */
     public function create($data)
     {
-        $interfaceDescriptor = new InterfaceDescriptor();
+        $interfaceDescriptor = new Interface_();
```

Comment:
I take it that the Interface_ element should still be refactored since it does not receive any constructor arguments (such as the FQSEN)?